docs/Maintainers/New-Season.md

```diff
@@ -1,3 +1,7 @@
+## Pre-Competition Season
+
```

Comment:
nit: Let's have a `Pre-Competition Season` h2 and then underneath an `Update Component OPRs` h3